site/en/codelabs/openthread-border-router-ipv6-multicast/index.lab.md

```diff
@@ -119,7 +119,7 @@ $ rm -rf build
 $ script/build nrf52840 USB_trans -DOT_MLR=ON
 ```
 
-Continue with the [Build a Thread network with nRF52840 boards and OpenThread codelab](https://openthread.io/codelabs/openthread-hardware#4) as written. After the end device is flashed with the CLI image, follow [Join the second node to the Thread network](https://openthread.io/guides/border-router/docker/test-connectivity#join-the-second-node-to-the-thread-network) to add the Thread device to the Thread network. Repeat for the second Thread end device.
+Continue with the [Build a Thread network with nRF52840 boards and OpenThread codelab](https://openthread.io/codelabs/openthread-hardware#4) as written. After the end device is flashed with the CLI image, follow [Join the second node to the Thread network](https://openthread.io/guides/border-router/form-network#join-the-second-node-to-the-thread-network) to add the Thread device to the Thread network. Repeat for the second Thread end device.
```

Comment:
I noticed the CLI reference link at the end of this codelab (and the main OTBR one you haven't edited) go to https://github.com/openthread/openthread/blob/main/src/cli/README.md#openthread-cli-reference

Should we update them to be the one on the website?  https://openthread.io/reference/cli/commands

The nat64 codelab uses the openthread.io link